canonical-websites/snapcraft.io
# Weekly active devices updates

Fixes https://github.com/canonical-websites/snapcraft.io/issues/291 and https://github.com/canonical-websites/snapcraft.io/issues/290 (to some extent, colours still need work)
should also fix https://github.com/canonical-websites/snapcraft.io/issues/289 (seemes to be JS error, not a map specific error)

# Work done

- Defended against an error when a label isn't present for some reason.
- Removed the 'Year' option for Weekly active devices.
- Combined non-Ubuntu OS versions (will probably need to be revised in the future).
- Sorted Weekly active device data in an attempt to make the data easier to read.
- Ensured axis labels are visible.
- Sorted tooltip data.

# QA

- Pull this branch
- `./run` and visit http://0.0.0.0:8004/account or http://snapcraft.io-pr-303.run.demo.haus/account
- Click through to the market page for a snap (with data)
- Ensure everything looks as expected and the tooltip data is sorted appropriately
- +1 this PR.

# Screenshot

![screenshot from 2018-02-14 12-15-04](https://user-images.githubusercontent.com/479384/36203836-e0b0c37a-1180-11e8-8007-cc3ea79d06d1.png)
